### PR TITLE
Create Welcome panel in rn_inspector

### PR DESCRIPTION
**Pull request by huntie**
## Summary

> This PR is based on (and squashes) @motiz88's starting point: https://github.com/motiz88/rn-chrome-devtools-frontend/pull/5/commits/1557bd7b196d4e7bd27370e775ee26243579b2a0.

Adds a minimal "Welcome" panel behind the experiment flag `REACT_NATIVE_SPECIFIC_UI`. 

## Test plan

The appearance of this panel is intended to be a simple splash screen. It adheres to Chrome DevTools UI patterns (fonts and colours), and may move to a collapsible panel later.

**Light appearance**

Screenshot 2023-07-24 at 15 54 19

**Dark appearance**

Screenshot 2023-07-24 at 15 54 47

**Accessibility (Lighthouse)**

image

### CODE REVIEW

File: config/gni/devtools_grd_files.gni [hunk 1]
```diff
@@ -190,6 +190,7 @@ grd_files_release_sources = [
   "front_end/Images/review.svg",
   "front_end/Images/right-panel-close.svg",
   "front_end/Images/right-panel-open.svg",
+  "front_end/Images/rnIcon.png",
```
Comment (motiz88) [inline config/gni/devtools_grd_files.gni]: Let's maybe put all RN-specific images in a subdirectory for ease of maintenance?

File: front_end/entrypoints/rn_inspector/rn_inspector.ts [hunk 2]
```diff
@@ -10,10 +10,21 @@ import '../inspector_main/inspector_main-meta.js';
...
 import '../../panels/timeline/timeline-meta.js';
+import '../../panels/rn_welcome/rn_welcome-meta.js';
 
 import * as Root from '../../core/root/root.js';
 import * as Main from '../main/main.js';
 
+Root.Runtime.experiments.register(
+  Root.Runtime.ExperimentName.REACT_NATIVE_SPECIFIC_UI,
+  'Show React Native-specific UI',
+  false,
```
Comment (motiz88) [inline front_end/entrypoints/rn_inspector/rn_inspector.ts]: I've already forgotten what this boolean argument means, so let's add a comment.

```suggestion
  /* unstable */ false,
```

File: config/gni/devtools_grd_files.gni [hunk 3]
```diff
@@ -177,6 +177,7 @@ grd_files_release_sources = [
   "front_end/Images/popup.svg",
   "front_end/Images/preview_feature_video_thumbnail.svg",
   "front_end/Images/profile.svg",
+  "front_end/Images/react_native/welcomeIcon.png",
```
Comment (huntie) [inline config/gni/devtools_grd_files.gni:180]: Actually perhaps these could be scoped by panel (`rn_welcome/`) 🤔

File: front_end/panels/rn_welcome/RNWelcome.ts [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+import * as UI from '../../ui/legacy/legacy.js';
+import * as i18n from '../../core/i18n/i18n.js';
+
+import rnWelcomeStyles from './rnWelcome.css.js';
+import * as LitHtml from '../../ui/lit-html/lit-html.js';
+
+const UIStrings = {
+  /** @description The name of the debugging product */
+  debuggerBrandName: 'Chrome DevTools for React Native',
```
Comment (motiz88) [inline front_end/panels/rn_welcome/RNWelcome.ts]: My latest thinking is we should go with "React Native JS Inspector" as a stopgap, until we can call it React Native DevTools or DevTools for React Native (with the future inclusion of React DevTools).

```suggestion
  debuggerBrandName: 'React Native JS Inspector',
```

**Pull request merged**